config.json

```diff
@@ -2,10 +2,6 @@
   "language": "C#",
   "active": true,
   "blurb": "C# is a modern, object-oriented language with lots of great features, such as type-inference and async/await. The tooling is excellent, and there is extensive, well-written documentation.",
-  "online_editor": {
```

Comment:
@ErikSchierboom The PowerShell script made these changes.  I'm not sure if I should leave it or manually remove it.  Just let me know.

Reply:
Ah yes, the PowerShell script is still a bit outdated. This is a side-effect of us working on Exercism v3. The key should not be removed.